airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/conftest.py

```diff
@@ -148,11 +143,22 @@ def connector_spec_fixture(connector_spec_path) -> Optional[ConnectorSpecificati
         return None
 
 
+@pytest.fixture(scope="session")
+def anyio_backend():
+    return "asyncio"
+
+
+@pytest.fixture(scope="session")
+async def dagger_client(anyio_backend):
```

Comment:
nit: docstring about dagger could be usefull as this might be the first exposure for a dev in the CAT tests where they have to care about dagger